docs/api-docs/getting-started/api-rate-limits.mdx

```diff
@@ -0,0 +1,115 @@
+
+<Callout type="info">
+  BigCommerce recommends adhering to industry-standard rate limits when developing.
+</Callout>
+
+## How limits work
+
+Each API request consumes one quota unit. The system rejects later requests once it reaches the quota. This continues until the quota is refreshed.
+
+All apps accessing the store share the store's quota. This prevents a single app from using the entire quota. The available quota for an app adjusts as other clients make or stop requests.
+
+## Rate limits by plan type
+
+Apps using OAuth are rate-limited based on a quota that's refreshed every few seconds. The maximum quota varies by store plan and requested resources.
+
+| Plan Type| Quota |
+|:------------------|:-------|
+| Enterprise and Enterprise-Test | by plan and [resource](https://support.bigcommerce.com/s/article/Platform-Limits#storelimits) |
+| Other sandboxes (Dev/Partner/Employee) | by [resource](https://support.bigcommerce.com/s/article/Platform-Limits#storelimits) |
+| Pro | 60k per hour (450 / 30sec) |
+| Plus & Standard | 20k per hour (150 / 30sec) |
+
+### Unlimited rate plan
+
+The Unlimited rate plan is for some BigCommerce Enterprise clients. It has no request rate limits. But, it may face limits from physical infrastructure. These limits may cap the maximum request throughput for a resource.
+
+ For more on resource constraints, see [object-related limits (Help Center)](https://support.bigcommerce.com/s/article/Platform-Limits#storelimits).
+
+<Callout type="warning">
+  BigCommerce can limit excessive API use to keep the platform stable. This is per our [Terms of Service](https://www.bigcommerce.com/terms/api-terms/).
+</Callout>
+
+## Best practices
+
+Manage API rate limits by following these crucial guidelines for success. 
+
+Process rate limit status codes accurately, distribute request loads evenly, store results,  and repeat requests strategically to boost app performance and prevent outages.
+
+### Handle rate limit status codes
+
+<Callout type="info">
+  As of January 2022, BigCommerce header field names are case-insensitive. Learn more about BigCommerce [response header conventions](/docs/start/about#response-headers).
+</Callout>
+
+API response headers show your position in the rate-limiting algorithm:
+
+```http filename="Standard plan example: Rate limit headers"
+X-Rate-Limit-Time-Window-Ms: 30000
+X-Rate-Limit-Time-Reset-Ms: 15000
+X-Rate-Limit-Requests-Quota: 150
+X-Rate-Limit-Requests-Left: 35
+```
+
+#### Seeing a `429` response?
+
+On receiving a `429` response, pause further requests until the quota refreshes:
+
+```http filename="Example: 429 response" showLineNumbers copy
+ HTTP/1.1 429 Too Many Requests
+ Date: Mon, 03 Feb 2022 20:36:00 GMT
+ Content-Type: application/json
+ X-Rate-Limit-Time-Reset-Ms: 15000
+```
+
+Use the `X-Rate-Limit-Time-Reset-MS` header to wait before retrying:
```

Comment:
WDYT if we add also some info about the other scenario, for adaptive rate limiting we do not provide back the headers with the request quota information, it is just 429 error